planning_domain:
  # Robot grippers
  vacuum_gripper: 'right arm'
  claw_gripper: 'left arm'
  
  # Table positions
  a_center_x: 0.20
  a_center_y: 0.25
  a_center_z: 0.25
  
  b_center_x: 0.20
  b_center_y: 0.25
  b_center_z: 0.25

  c_center_x: 0.20
  c_center_y: 0.25
  c_center_z: 0.25

  d_center_x: 0.20
  d_center_y: 0.25
  d_center_z: 0.25

  # Table object dimension ratio
  cube_x: 1.0
  cube_y: 1.0
  cube_z: 1.0

  plate_x: 1.0
  plate_y: 1.0
  plate_z: 0.25

  tower_x: 1.0
  tower_y: 1.0
  tower_z: 3.0

  rectangle_x: 1.0
  rectangle_y: 1.0
  rectangle_z: 0.25
